apps/address-service/admin-ui/index.js

```diff
@@ -1,5 +1,10 @@
 import Logo from '@app/address-service/admin-ui/logo'
+import React from 'react'
 
 export default {
     logo: Logo,
+    pages: () => {
+        window.React = React
```

Comment:
Where it's been used?

Reply:
This is neessary for make toast messages work within admin inteface.

Reply:
Could you please write comments, because this is implicit logic, not directly trackable